### PR TITLE
Plane: add param Q_TILT_BIAS_CH for RC control of motor tilt forward bias

### PR DESCRIPTION
**Pull request by kd0aij**
this lets you use a knob or slider to tilt the motors forward for higher speed in VTOL modes
Has no effect on FW modes with the motors fully forward.

### CODE REVIEW

File: ArduPlane/tiltrotor.cpp [hunk 1]
```diff
@@ -68,7 +68,7 @@ void QuadPlane::tiltrotor_continuous_update(void)
         max_change = tilt_max_change(false);
         
         float new_throttle = constrain_float(SRV_Channels::get_output_scaled(SRV_Channel::k_throttle)*0.01, 0, 1);
-        if (tilt.current_tilt < 1) {
+        if (tilt_get_biased_tilt() < 1) {
```
Comment (IamPete1) [inline ArduPlane/tiltrotor.cpp]: not sure if we should change this one, seems its for forward flight only.

**Comment by IamPete1** [comment]
loads of cool stuff this week! looks good to me

File: ArduPlane/tiltrotor.cpp (hunk 1 above)
Comment (kd0aij) [reply ArduPlane/tiltrotor.cpp]: with the bias applied in tilt_slew() the transition to FW will be start from the biased tilt angle and the motors will be fully forward when tilt_get_biased_tilt==1 and current_tilt is (1-bias) (although current_tilt will continue to increase till it hits 1, and biased_tilt is constrained to [0-1]
So I think this is correct.

Comment (IamPete1) [reply ArduPlane/tiltrotor.cpp]: ah, i see what you mean, there would be a step as the bias is removed.

**Comment by IamPete1** [comment]
should add Qacro to this if i think while your updating tiltrotor stuff https://github.com/ArduPilot/ardupilot/blob/415042e5576cfe3f93d16993075a7d806e98730e/ArduPlane/tiltrotor.cpp#L112

**Comment by kd0aij** [comment]
@IamPete1 Thanks for catching that; I added qacro to that zero-tilt case as suggested. Doesn't appear to change behavior in qacro, which is why I didn't notice it in testing.

**Comment by kd0aij** [comment]
tested in SITL and flight tested on E-Flite Convergence

**Pull request closed without merging**